django/db/models/fields/related_descriptors.py

```diff
@@ -259,6 +259,27 @@ def __set__(self, instance, value):
             setattr(value, self.field.remote_field.get_cache_name(), instance)
 
 
+class ForwardOneToOneDescriptor(ForwardManyToOneDescriptor):
+
+    def get_object(self, instance):
+        if self.field.remote_field.parent_link:
+            deferred = instance.get_deferred_fields()
+            # Because it's a parent link, we have all the data
```

Comment:
try to avoid "we", e.g. "Because it's a parent link, all the data is available in the instance, ...